app/views/search/opensearch.xml

```diff
@@ -2,5 +2,5 @@
 <OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/" xmlns:moz="http://www.mozilla.org/2006/browser/search/">
     <ShortName>GOV.UK</ShortName>
     <Description>Search GOV.UK - The best place to find government services and information.</Description>
-    <Url type="text/html" template="https://www.gov.uk/search?q={searchTerms}"/>
+    <Url type="text/html" template="https://www.gov.uk/search/all?keywords={searchTerms}"/>
```

Comment:
Should this be `all-content`?

Reply:
Actually, no you're right - it turns out I missed something in the spec for https://trello.com/c/SjudMnoT/499-update-urls-across-all-7-finders 😄 thanks!